apps/user-office-backend/src/datasources/postgres/AdminDataSource.ts

```diff
@@ -67,19 +69,14 @@ export default class PostgresAdminDataSource implements AdminDataSource {
       .returning('*');
 
     if (!institutionRecord) {
-      throw new Error(`Could not update institution with id:${institution.id}`);
+      return null;
```

Comment:
We also do it here.

Reply:
It is the same for this one I just transferred the logic for error handling in the controller so to say.